### DISCUSSION TITLE
Zettlr Mobile App

### DISCUSSION THREAD

**Discussion by mubashirjamali101** [Zettlr/Zettlr#3535] [General]
# Hi!
## Thank you for making this app and for contributing in this app.
Hope you guys are doing good, I know this idea is already been thrown a few times. But, is there a plan for Zettlr mobile app? I have used Notion, Obsidian etc, but simplicity and minimalism of Zettlr keeps me using it. I wonder if there was a mobile app for Zettlr...

**Thanks a lot**
Mubashir

**Comment by 87113** [comment]
Hi, I don't know if Zettler is going to be on mobile, but I recently discover another app Zettel : https://znotes.thedoc.eu.org/ 
I like it very much. :)

**Reply by skoufaki** [reply]
Unfortunately it's not FLOSS (free libre open source), but proprietary. :(

https://github.com/damionx7/Zettel-Notes-Documentation/issues/1

**Comment by asmaier** [comment]
The problem with developing a mobile app is that Zettlr depends on LaTeX and pandoc. Both are not available on mobile platforms. However LaTeX for PDF output could be replaced by Typst. And since v3.9 pandoc can be compiled to web assembly and run in the browser, see https://pandoc.org/app/ . I think both developments now would make it possible to develop a Zettlr app, that also runs on mobile.

**Reply by mubashirjamali101** [reply]
I also heard that LaTeX could be implemented via KaTeX, not sure though.

**Comment by BabakFarshchian** [comment]
Take a look at how I use Zettlr with Zettel Notes on Android (might be a bit outdated):

https://blog.babak.no/2023/02/04/how-to-almost-use-zettlr-on-android-devices/
